app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -79,169 +109,103 @@ form {
     }
   }
 
-  /* Nested Fieldsets and Legends */
+  /* Nested Fieldsets and Legends (radio, check boxes and date/time inputs use nested fieldsets) */
```

Comment:
Use `//` comments everywhere<br>Each selector in a comma sequence should be on its own line